packages/blockchain/src/consensus/ethash.ts

```diff
@@ -45,7 +44,7 @@ export class EthashConsensus implements Consensus {
   public async genesisInit(): Promise<void> {}
   public async setup({ blockchain }: ConsensusOptions): Promise<void> {
     this.blockchain = blockchain
-    this._ethash = new Ethash(this.blockchain.db as unknown as EthashCacheDB)
+    this._ethash = new Ethash(this.blockchain.db as any)
```

Comment:
Not for this PR, but should we really take this occasion and loosen the binding between blockchain and the Ethash class/package? So eventually make this an optional dependency? 🤔

I think this is not an easy side-shot though but needs some additional design thoughts, regarding consensus instantiation (eventually) on mainnet, thinks like that.

But at the end should not be that complicated/invasive and I guess it's maybe/likely the time for that.